src/box/lua/config/instance_config.lua

```diff
@@ -2211,6 +2211,12 @@ return schema.new('instance_config', schema.record({
             default = 10,
         }),
     })),
+    connpool = schema.record({
+        idle_timeout = schema.scalar({
+            type = 'number',
+            default = 60,
+        }),
+    }),
```

Comment:
@grafin wrote:

> I just like to have an ability to turn this function off, by setting timeout to zero or a negative value. Maybe we should add it? Just in case if I have a small cluster, and don't really care for the opened file descriptors.

I would discuss it in a thread.